x/tokenomics/keeper/token_logic_modules_test.go

```diff
@@ -76,11 +76,10 @@ func TestProcessTokenLogicModules_TLMBurnEqualsMint_Valid(t *testing.T) {
 	require.NoError(t, err)
 	// TODO_TECHDEBT: Setting inflation to zero so we are testing the BurnEqualsMint logic exclusively.
 	// Once it is a governance param, update it using the keeper above.
-	prevInflationValue := tlm.GlobalInflationPerClaim
-	tlm.GlobalInflationPerClaim = 0
-	t.Cleanup(func() {
-		tlm.GlobalInflationPerClaim = prevInflationValue
-	})
+	tokenomicsParams := keepers.Keeper.GetParams(ctx)
+	tokenomicsParams.GlobalInflationPerClaim = 0
```

Comment:
This won't be possible anymore once the validation function rejects 0; see: https://github.com/pokt-network/poktroll/pull/951#discussion_r1853038203.